server/index.js

```diff
@@ -1,14 +1,40 @@
 const express = require('express');
 const path = require('path');
 const chalk = require('chalk');
+const db = require('../relatedData.js');
 const app = express();
 
 const port = 3001;
 
 app.use(express.static(path.join(__dirname, '../public')));
 
-app.get('/', (req, res) => {
-  res.send('Hi');
+app.get('/relatedTracks/:song', (req, res) => {
+  // console.log(chalk.bgWhite.black(req.params.song));
+  db.findTrack(req.params.song, (err, data) => {
+    if (err) {
+      console.log(chalk.red(`Can't find track: `, err));
+    } else {
+      let relatedInfo = [];
+      for (let i = 0; i < data.related.length; i++) {
```

Comment:
We talked during office hours about dangerous property look ups on an object. However, I didn't give you a solution.

https://lodash.com/docs/4.17.15#get

never fear a property look up again. Set sensible default